SignalServiceKit/src/Storage/OWSSessionStorage.h

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSStorage.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// TODO: Migrate data from primary data store.
+// TODO: Add "is migrated flag".
+// TODO: Check "is migrated flag" before accessing session state.
+// TODO: Close database in background.
+// TODO: Use background task around transactions.
```

Comment:
Some of the immediate TODOs - some are addressed (or mostly addressed) by this PR.  This list is updated in next PR.